pandas/io/tests/test_sql.py

```diff
@@ -236,7 +236,7 @@ def _close_conn(self):
         pass
 
 
-class PandasSQLTest(unittest.TestCase):
+class PandasSQLTest(object):
```

Comment:
@TomAugspurger A gentle reminder here - in my experience pytest won't run this class, because of its name. Not 100% sure, but at least worth double-checking it's run.